.github/ISSUE_TEMPLATE/collaboration-cycle-request.yml

```diff
@@ -68,15 +70,15 @@ body:
     id: start-date
     attributes:
       label: When did/will you start working on this product/feature?
-    validations:
+        validations:
```

Comment:
Looks like an extra indentation got added here. GitHub's form engine will choke on that.

Reply:
Thanks for catching/flagging!